resources/views/maps/form/_markers.blade.php

```diff
@@ -150,37 +150,33 @@ class="fa-solid fa-map"></i> {{ __('maps.actions.explore') }}</a>
     <div class="modal fade" id="marker-modal" role="dialog" aria-labelledby="deleteConfirmLabel">
         <div class="modal-dialog modal-lg" role="document">
             <div class="modal-content">
-                <div class="panel panel-default">
-                    <div class="panel-heading">
-                        <button type="button" class="close" data-dismiss="modal"
-                            aria-label="{{ __('crud.delete_modal.close') }}"><span
-                                aria-hidden="true">&times;</span></button>
-                        <h4>
-                            {{ __('maps/markers.create.title', ['name' => $model->name]) }}
-                        </h4>
-                    </div>
-                    <div class="panel-body">
-
-                        {!! Form::open([
-    'route' => ['maps.map_markers.store', $model],
-    'method' => 'POST',
-    //'enctype' => 'multipart/form-data',
-    //'id' => 'map-marker-new-form'
-    'class' => 'ajax-subform',
-]) !!}
-                        @include('maps.markers._form', ['model' => null, 'map' => $model, 'activeTab' => 1, 'dropdownParent' => '#marker-modal'])
-
-                        <div class="form-group">
-                            <div class="pull-left">
-                                @include('partials.footer_cancel', ['ajax' => 1])
-                            </div>
-                            <div class="pull-right">
-                                @include('maps.markers._actions')
-                            </div>
+                <div class="panel-heading">
```

Comment:
![image](https://user-images.githubusercontent.com/3263440/197631952-052f96f1-939a-4b07-a42c-f149441b3fd8.png)
`panel-*` classes are still wrong when creating a marker in an modal, it should be "modal-body" and "modal-footer".